brave/browser-laptop
# Adds captcha to UGP flow

Resolves #14188 

## Submitter Checklist:


## Test Plan:
- clean profile
- start browser with staging flag
- enable payments
- claim promotion
- verify that you can see captcha overlay
- solve (and try to miss) the captcha
- make sure that success overlay is displayed
- close overlay and make sure that you received 10BAT

## Reviewer Checklist:


Tests
